[2025-02-21T08:24:11.008+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_consumer_dag.consume_kafka_messages manual__2025-02-21T08:24:09.855055+00:00 [queued]>
[2025-02-21T08:24:11.023+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_consumer_dag.consume_kafka_messages manual__2025-02-21T08:24:09.855055+00:00 [queued]>
[2025-02-21T08:24:11.023+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-21T08:24:11.042+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): consume_kafka_messages> on 2025-02-21 08:24:09.855055+00:00
[2025-02-21T08:24:11.050+0000] {standard_task_runner.py:57} INFO - Started process 456 to run task
[2025-02-21T08:24:11.054+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_consumer_dag', 'consume_kafka_messages', 'manual__2025-02-21T08:24:09.855055+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/consume.py', '--cfg-path', '/tmp/tmpahh80grx']
[2025-02-21T08:24:11.055+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask consume_kafka_messages
[2025-02-21T08:24:11.119+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_consumer_dag.consume_kafka_messages manual__2025-02-21T08:24:09.855055+00:00 [running]> on host c0c1f1afe7d6
[2025-02-21T08:24:11.208+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_consumer_dag' AIRFLOW_CTX_TASK_ID='consume_kafka_messages' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T08:24:09.855055+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-21T08:24:09.855055+00:00'
[2025-02-21T08:24:11.212+0000] {logging_mixin.py:154} INFO - Consuming messages from topic 'from_mysql_transaction'...
[2025-02-21T08:24:21.215+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-21T08:24:21.230+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_consumer_dag, task_id=consume_kafka_messages, execution_date=20250221T082409, start_date=20250221T082411, end_date=20250221T082421
[2025-02-21T08:24:21.276+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-21T08:24:21.297+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
